### PR TITLE
add code coverage and refactor tests folder

### PR DESCRIPTION
**Pull request by andrewdamelio**
### add code coverage and refactor tests folder
 - addresses https://github.com/rangle/react-redux-starter/issues/26

@neilff @SethDavenport let me know if you see a way to simplify and optimize this... I felt like i had to jump through a lot of weird loops for the coverage. :fearful:

### CODE REVIEW

File: tests/entry.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import requireHacker from 'require-hacker';
+
+[ 'css',
+  'svg',
```
Comment (neilff) [inline tests/entry.js]: should we add additional files types, e.g. `.png',  `.gif',  `.jpg', etc.

**Comment by neilff** [comment]
looks good other than the one note

File: tests/entry.js (hunk 1 above)
Comment (andrewdamelio) [inline tests/entry.js]: will do

**Comment by benmliang** [comment]
:+1: This (`require-hacker`) is an elegant solution that solves the a super annoying problem I had spend hours to fix :100:

**Comment by andrewdamelio** [comment]
@SethDavenport any feedback?

**Comment by SethDavenport** [comment]
Looks good other than conflicts.

@rbrander : might be worth it for priceline.

File: tests/component/ui/Modal.test.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import jsdom from 'jsdom';
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-import Modal from './Modal';
+import Modal from '../../../src/components/ui/Modal';
```
Comment (JaKXz) [inline tests/component/ui/Modal.test.js]: perhaps using `path.resolve` will make life easier in tests?

**Pull request merged**